### ISSUE TITLE
Scrollbar in terminal

### ISSUE THREAD

**Issue by Starix36** [platformio/platformio-atom-ide-terminal#800]
- [ ] I have reset Atom to defaults prior to submitting report.
- [ ] I have not reset Atom to defaults prior to submitting report.
### Steps to reproduce

1.
2.
3.

**Expected behavior:** 

**Actual behavior:** 

**Reproduces how often:** 

### Versions

```
> atom --version

```
```
> apm --version

```
OS name and version:
Platformio-ide-terminal version:

**Comment by the-j0k3r** [comment]
duplicate of https://github.com/platformio/platformio-atom-ide-terminal/issues/9

Also this depends on #426 since `term.js` does not support scrollbars and is no longer developed.

Please dont use capitals next time.
